src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -59,6 +64,40 @@ public static bool IsBase64UrlEncodingChar(this char ch)
                ch == '_';
     }
 
+    public static bool TryValidateCommonAnnotatedKey(byte[] key,
+                                                     string base64EncodedSignature)
+    {
+        if (key == null || 
+            (key.Length != StandardCommonAnnotatedKeySizeInBytes && key.Length != LongFormCommonAnnotatedKeySizeInBytes))
+        {
+            return false;
+        }
+
+        bool longForm = key.Length == LongFormCommonAnnotatedKeySizeInBytes;
+
+        ulong checksumSeed = VersionTwoChecksumSeed;
+
+        byte[] bytesToChecksum = new byte[60];
+        Array.Copy(key, bytesToChecksum, bytesToChecksum.Length);
+
+        
+        int checksum = Marvin.ComputeHash32(bytesToChecksum, checksumSeed, 0, bytesToChecksum.Length);
```

Comment:
this apparently simply change is actually disruptive because other API work directly against byte[] rather than IEnumerable<byte>. that probably should be changed but it's API breaking so not doing it now. :)

The code you proposed to change is just a one-liner, so I think it's ok to retain, though arguably not as readable/elegant as your suggestion.